lib/http.js

```diff
@@ -378,6 +392,12 @@ async function _handleOperation({req, expectedRootCapability, keystoreId}) {
   const result = await validateOperation({
     url, method, headers,
     operation,
+    // FIXME: this value is passed down to http-signature-zcap-verify
+    // where the `host` header is compared to the expected value:
+    // eslint-disable-next-line max-len
+    // https://github.com/digitalbazaar/http-signature-zcap-verify/blob/master/main.js#L82-L83
+    // if dispensing with `allowedHost`, does this simply become the local
+    // server config.server.host or domain?
```

Comment:
Yes, this should be the host value for the KMS server itself (it is no longer for some application that is using the KMS system).